hbase-it/src/test/java/org/apache/hadoop/hbase/IntegrationTestRegionReplicaReplication.java

```diff
@@ -94,7 +95,7 @@ public void setConf(Configuration conf) {
       String.format("%s.%s", TEST_NAME, LoadTestTool.OPT_COLUMN_FAMILIES),
       StringUtils.join(",", DEFAULT_COLUMN_FAMILIES));
 
-    conf.setBoolean("hbase.table.sanity.checks", true);
+    conf.setBoolean(TableDescriptorChecker.TABLE_SANITY_CHECKS, true);
```

Comment:
Is this different from other `conf.setBoolean("hbase.table.sanity.checks", true)` which is removed from other tests like TestAvoidCellReferencesIntoShippedBlocks based on the default value of TableDescriptorChecker.TABLE_SANITY_CHECKS?

Reply:
This is in moudle hbase-it. And this may be runed by user directly and with user's hbase-site.xml? So i didn't remove this.